internal/datatransfer/s3/sync.go

```diff
@@ -53,6 +53,7 @@ func NewSync(s3Config models.S3StorageConfiguration) (*Sync, error) {
 
 	sync.endpoint = fmt.Sprintf("https://%s:%d", s3Config.BucketHost, s3Config.BucketPort)
 	sync.bucketName = s3Config.BucketName
+	sync.bucketRegion = s3Config.BucketRegion
```

Comment:
It was there a few PRs ago https://github.com/jakub-dzon/k4e-device-worker/pull/42/files#diff-5a3f35566e47eb15cffc1c02a0b18f6bbbf48883663884e699ed09ca12942ca7L43